.kitchen.yml

```diff
@@ -15,13 +15,16 @@ platforms:
   - name: amazon-linux
     driver_config:
       box: mvbcoding/awslinux
-  - name: centos-6
   - name: centos-7
+  - name: redhat-7
```

Comment:
centos-7 should/does cover redhat-7. We don't tend to test using this as it isn't available as a dokken or Vagrant image

Reply:
It shooooould, but I've had some issues with my testing on the EC2 provider with package inconsistencies between variants. RHEL-7 fails to install libdnet and libdnet-devel, where CentOS 7 is perfectly happy.

I take the point about Vagrant though, I haven't been checking that it would work since I broke my work desktop with too many kitchen instances running at once.